tasks/Centos.yml

```diff
@@ -0,0 +1 @@
+Redhat.yml
```

Comment:
I'm not 100% sure but believe it should be `RedHat` ?
```suggestion
RedHat.yml
```

Reply:
[Link](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html#ansible-facts-os-family) I believe you are correct 👍